pytorch_lightning/trainer/trainer.py

```diff
@@ -635,17 +635,27 @@ def data_parallel(self) -> bool:
         return self.use_dp or self.use_ddp or self.use_ddp2
 
     @property
-    def training_tqdm_dict(self) -> dict:
-        """Read-only for tqdm metrics.
-        :return:
-        """
+    def progress_bar_dict(self) -> dict:
+        """ Read-only for progress bar metrics. """
         ref_model = self.model if not self.data_parallel else self.model.module
+        return dict(**ref_model.get_progress_bar_dict(), **self.progress_bar_metrics)
 
-        return dict(**ref_model.get_tqdm_dict(), **self.tqdm_metrics)
+    @property
```

Comment:
move this to `deprecated_api.py`

Reply:
to 0.9 or later?

Reply:
we use waiting 0.2 so deprecated in 0.7.x will be removed 0.9

Reply:
and it shall be tested as #1470
so @williamFalcon could you merge it?

Reply:
ok I will move it to 0.9 then

Reply:
@Borda is it correct now? anything else I missed for deprecated api?